Manuscript/Text/02F_mutein.md

```diff
@@ -1,3 +1,9 @@
 ## Model Accurately Captures Cell Type-Specific Response to IL-2 Muteins
 
 ![**Model and tensor factorization predicts and decodes cell-type specific responses to IL-2 muteins.** A) IL-2Rα and IL-2Rβ/γ~c~ dissociation constants for panel including wild-type IL-2 and IL-2 muteins. B-C) Predicted and measured T~reg~ activation specificity compared to NK (B) and T helper (C) cells. D) Partial derivatives of T~reg~ activation specificity compared to NK and T helper cells with respect to each surface and endosomal reverse binding rate constant. E) T~reg~ activation specificity with respect to NK and T helper cells as a function of IL-2Rβ/γ~c~ binding affinity for ligands with wild type and reduced IL-2Rα affinity. Specificity values are shown for cells exposed to a cytokine concentration of 38 pM. F) Experimentally measured and model-predicted EC~50~s of dose responses across IL-2 muteins and all 10 cell types. EC~50~s are shown for 1 hr time point. G-J) Tensor factorization of experimentally measured cellular activation values for wild-type IL-2/15, and IL-2 muteins. Component values versus ligand (G), cell type (H), time (I), and cytokine concentration (J). ](./Figures/figure6.svg){#fig:mutFac}
```

Comment:
I mention this to everyone—suggest running this through grammarly.com and looking through the suggestions. Call it a language linter. :-)